README.md

```diff
@@ -31,7 +33,22 @@ globParent('path/**/*'); // 'path'
 globParent('path/foo/bar.js'); // 'path/foo'
 globParent('path/foo/'); // 'path/foo'
 globParent('path/foo'); // 'path' (see issue #3 for details)
+```
+
+## Escaping
+
+The following characters have special significance in glob patterns and must be escaped if you want them to be treated as regular path characters:
 
+- `?` (question mark)
+- `*` (star)
+- `[` (opening bracket)
+- `]` (closing bracket)
```

Comment:
If you can update this PR with those other commits from my continuation branch and add the other escape-able chars to this list then I think we'll be ready to wrap this up.